src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -36,7 +36,7 @@ const COVERAGE_FILE: &str = "coverage.json";
 const SOURCE_COVERAGE_FILE: &str = "source-coverage.json";
 const MODULE_CACHE_FILE: &str = "module-cache.json";
 
-const DEFAULT_TARGET_TIMEOUT: Duration = Duration::from_secs(5);
+const DEFAULT_TARGET_TIMEOUT: Duration = Duration::from_secs(120);
```

Comment:
If a user sets their own `target_timeout` that is less than the time required for the initial coverage recording (and associated cold module / symbol cache misses), then this generous window will not help.

Consider, at this constant's [use site](https://github.com/microsoft/onefuzz/blob/cafba18c8500bce241a7480880ae6f2e9717a31a/src/agent/onefuzz-task/src/tasks/coverage/generic.rs#L59-L63), either adding a `TIMEOUT_BUFFER` value, setting a minimum, or applying a multiplier (perhaps with a max?).

Reply:
I'll won't address it here but I'll open an issue to track.